waspc/e2e-test/GoldenTest.hs

```diff
@@ -7,11 +7,26 @@ where
 
 import Common (getTestOutputsDir)
 import Control.Monad (filterM)
-import Data.List (sort)
+import Data.Aeson (Value, decode)
+import Data.Aeson.Encode.Pretty
+  ( Config (Config, confCompare, confIndent, confNumFormat, confTrailingNewline),
```

Comment:
I would maybe import this stuff qualified, as I find it a bit hard in the code to figure out where it is coming from, when reading it.

Reply:
Yep, can do 👍🏻